api/swagger.yaml

```diff
@@ -5904,6 +5904,10 @@ paths:
           description: "The volume was created successfully"
           schema:
             $ref: "#/definitions/Volume"
+        304:
+          description: "The volume has already been created"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
```

Comment:
Ah yes, 3xx only means "additional actions may be required" so typically it is not considered as error for restful APIs. For example, 301 for "Moved Permanently", 307 for "Redirect".

I checked the response it indeed does not return error code. So just updated the API swargger.yml. Thanks